dgraphtest/local_cluster.go

```diff
@@ -343,6 +347,28 @@ func (c *LocalCluster) killContainer(dc dnode) error {
 	return nil
 }
 
+func (c *LocalCluster) HealthCheckAlpha(id int) error {
```

Comment:
The way we should do this is, We should modify the `StartAlpha` and `StartZero` function to also perform the health check before returning. We should refactor the `HealthCheck` function on `*LocalCluster` type to be able to do the health check on a given alpha or zero. We should use the `dnode` interface as an argument to the health check function because it can call healthURL function independent of whether it is a zero or alpha.

Reply:
Done